src/Servers/Kestrel/perf/PlatformBenchmarks/PlatformBenchmarks.csproj

```diff
@@ -23,6 +23,11 @@
 
   <!-- These references are used when running on the Benchmarks Server -->
   <ItemGroup Condition="'$(BenchmarksTargetFramework)' != ''">
-    <Reference Include="Microsoft.AspNetCore.All" />
+    <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv" version="2.2.0" />
+    <KnownFrameworkReference 
+      Update="Microsoft.AspNetCore.App"
```

Comment:
@natemcmaster I remember you saying you didn't expect this to work for long. Is there a less fragile alternative?